CHANGELOG.rst

```diff
@@ -11,6 +11,7 @@ This document describes changes between each past release.
 - ``kinto --version`` was renamed ``kinto version``
 - ``ResourceChanged`` and ``AfterResourceChanged`` events now return
   ``old`` and ``new`` records for the ``delete`` action. (#751)
+- Redis backends are not parts of the core anymore. (#712)
```

Comment:
* Note: this is a breaking change (would lead to kinto 4.0).
* I suggest that we give some details. For example `In settings, replace kinto.core.cache.redis by kinto_redis.cache`
* I don't think we use the term `backend` for the listener, so some details about that would be required too I think
* typo : *are not part of*